tests/beaker_tests/syslog_settings/syslog_settings_provider_defaults.rb

```diff
@@ -60,70 +60,72 @@
 
 # @test_name [TestCase] Executes defaults testcase for syslog_settings Resource.
 test_name "TestCase :: #{testheader}" do
-  # @step [Step] Sets up switch for provider test.
-  step 'TestStep :: Setup switch for provider' do
-    # For deterministic results, make sure syslog_settings is set to
-    # seconds.
-    on(master, SyslogSettingLib.create_syslog_settings_manifest_seconds)
-    cmd_str = get_namespace_cmd(agent, PUPPET_BINPATH +
-      'agent -t', options)
-    on(agent, cmd_str, acceptable_exit_codes: [0, 2])
-
-    logger.info('Setup switch for provider')
-  end
+  if operating_system != 'ios_xr'
```

Comment:
Instead of wrapping the whole test in an 'if' block, I think you could just `raise_skip_exception if operating_system == 'ios_xr'`. Would make for a much smaller diff.